_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -75920,6 +75918,10 @@
 /obj/structure/disposalpipe/segment,
 /turf/open/floor/plating,
 /area/maintenance/starboard/secondary)
+"vZr" = (
+/obj/machinery/firealarm,
+/turf/closed/wall,
```

Comment:
don't place alarms inside of the wall.

Reply:
Hopefully fixed now, scream at me again if i did not.